airbyte-ci/connectors/metadata_service/lib/tests/fixtures/__init__.py

```diff
@@ -3,15 +3,26 @@
 from typing import List
 
 
+def list_all_paths_in_fixture_directory(folder_name: str) -> List[str]:
+    file_path = os.path.join(os.path.dirname(__file__), folder_name)
+    return [os.path.join(file_path, file_name) for file_name in os.listdir(file_path)]
```

Comment:
nit: I tend to prefer using Pathlib instead of os.path
```suggestion
    return Path( os.path.dirname(__file__) / folder_name).iterdir()
```